prow/images/kyma-integration/README.md

```diff
@@ -0,0 +1,38 @@
+# Kyma-integration images
+
+## Overview
+
+This folder contains the image with tools necessary for provisioning kyma-integrtion clusters.
+
+The image consists of:
+- Go
+- Dep
+- Helm
+- Gcloud
+- Az
+- Docker
+
+# Maintaning the image
+
+Google Cloud SDK comes with one default `kubectl` version and couple additional ones.
+See details [here](https://cloud.google.com/sdk/docs/release-notes#27600_2020-01-14).
+
+`CLUSTER_VERSION` variable is used to match `kubectl` version used in the image with the cluster version that is build by ProwJob pipeline.
+
+```
+ENV CLUSTER_VERSION=1.14
+RUN mv /google-cloud-sdk/bin/kubectl.${CLUSTER_VERSION} /google-cloud-sdk/bin/kubectl
+```
+
+**CAUTION** Each image is tagged with corresponding `kubectl` version. While updating the image please do not forget to adjust the tag. 
+```
```

Comment:
Add such sentence before the command:
```To adjust the tag on the updated Docker image, run:```

Reply:
It is not a command that you run separately. This line comes from Makefile and the only thing you have to do is to update the `k8s-1.14` tag.